webapp/components/test-run.html

```diff
@@ -49,7 +49,7 @@
 
     <div>
       <div><img src="/static/{{testRun.browser_name}}_64x64.png" /></div>
-      <div>{{displayName(testRun.browser_name)}} {{testRun.browser_version}}</div>
+      <div>{{displayName(testRun.browser_name)}} {{shortVersion()}}</div>
```

Comment:
The function `shortVersion` should probably depend on `testRun.browser_version` as a param so that if the `test-run` element is re-used (currently not the case) then the function's output value won't get stale.

Reply:
Shows what I know about Polymer. I've updated it, also explicitly depending on `browser_name` so that rendering of the minor version likewise respects changes.